README.md

```diff
@@ -1 +1,136 @@
-# leaderboard-app
\ No newline at end of file
+# Table of Contents
+
+- About the Project
+- Built With
+- Tech Stack
+- Key Features
+- Live Demo
+- Getting Started
+- Setup
+- Prerequisites
+- Install
+- WebPack
+- Run tests
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+---
+
+
+**[Leader-Board-App]** is used to display a Score of Participants.
+
+## 🛠 Built With
+
+### Tech Stack
+
+This project is built using HTML, CSS, and JavaScript.
+
+### Key Features
+
+- **[Leader-Board-App]**
+
+---
+
+## Live Demo
+
+This Project is not Deployed...
+
+---
+
+## Getting Started
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Make sure you have the following installed:
+
+- Node.js
+- git
+- Linters (Lighthouse, Webhint, and Stylelint)
+- A code editor
+
+Example command for installing prerequisites:
+npm install -g lighthouse
+npm install --save-dev hint@7.x stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd my-folder
+git clone https://github.com/MustafaSamiShirzad/leaderboader-app
+
+### Install
+
+Install this project with:
+
+git clone https://github.com/MustafaSamiShirzad/leaderboard-app
+
+### WebPack Plugins
+
+Before you proceed, make sure you have initialized npm using `npm init -y`.
+Next, install the required plugins:
+
+npm i -D html-webpack-plugin sass style-loader css-loader sass-loader
+
+Install Webpack Dev Server:
+
+Install Webpack Dev Server:
+
+npm run dev
+
+### Run tests
+
+To run tests, use the following commands:
+
+npx hint . # Used to check HTML errors
+npx stylelint "\*_/_.{css,scss}" # Used to check CSS errors
+npx eslint . # Used to check JavaScript errors
+
+After installing the project, run the following commands:
+
+npm i # or npm install to download all the required packages
+npm run dev # to open the project in your browser
+
+---
+
+## Authors
+
+**Mustafa Shirzad**
+
+- GitHub: [@githubhandle](https://github.com/MustafaSamiShirzad)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mustafa-shirzad-9ab766179/)
+
+---
+
+## Future Features
+
+- [ ] **[Will be integrating Api(application programming interface)]**
+
+---
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+
+---
+
+## Show your support
+
+If you like this project, please support it by reaching out to me via email at Mustafasami209@gmail.com
+
+---
+
+## Acknowledgments
+
+--- It is a solo project
+
+## 📝 License
+
+This project is [MIT licensed](./LICENSE).
```

Comment:
> - [ ] I have observed that the link to your MIT License is not functional. It is essential to address this issue promptly to ensure compliance and clarity regarding the licensing terms of your software or project.😊👍
> 
> ![Screenshot](https://github.com/MustafaSamiShirzad/leaderboard-app/assets/40009354/326e662b-b2ad-4737-9373-31431f5b6bad)
>

Reply:
I added the LICENCE, thanks for your remarks.